src/webob/acceptparse.py

```diff
@@ -407,25 +432,70 @@ def generator(value):
                 )
         return generator(value=value)
 
-    def _parse_and_normalize_offers(self, offers):
+    @classmethod
+    def parse_offer(cls, offer):
+        """
+        Parse an offer into its component parts.
+
+        :param offer: A media type or range in the format
+                      ``type/subtype[;params]``.
+        :return: A named tuple containing ``(*type*, *subtype*, *params*)``.
+
+                 | *params* is a list containing ``(*parameter name*, *value*)``
+                   values.
+
+                 | The result also supports ``is_range`` and ``specificity``
+                   properties. Specificity is a value from 1 to 4 where ``*/*``
+                   is 1, ``text/*`` is 2, ``text/html`` is 3 and
+                   ``text/html;charset=utf8`` is 4.
+
+        :raises ValueError: If the offer does not match the required format.
+
+        """
+        match = cls.media_type_compiled_re.match(offer)
+        if not match:
+            raise ValueError('Invalid value for an Accept offer.')
+
+        groups = match.groups()
+        offer_type, offer_subtype = groups[0].split('/')
+        offer_params = cls._parse_media_type_params(
+            media_type_params_segment=groups[1],
+        )
+        # offer_type, offer_subtype, offer_params, invalid, example
+        #    == *        == *           true         Y/N
+        #      N           N              N           N      a/b
+        #      N           N              Y           N      a/b;x=y
+        #      N           Y              N           N      a/*
+        #      N           Y              Y           Y      a/*;x=y
+        #      Y           N              N           Y      */b
+        #      Y           N              Y           Y      */b;x=y
+        #      Y           Y              N           N      */*
+        #      Y           Y              Y           Y      */*;x=y
+        # simplifies to (A and not B or B and C)
```

Comment:
Would this table be useful in the [narrative docs](https://docs.pylonsproject.org/projects/webob/en/latest/reference.html#accept-headers)?

I don't understand the "simplifies" comment. Does it pertain to the table's "example" column?

Reply:
Do A, B, and C map to the columns, the example, or something else?

Reply:
yes